### PR TITLE
Added support for small windows in the graphing calculator

### PR DESCRIPTION
**Pull request by rudyhuyn**
## Part of #338 

Add support for small windows in the graphing calculator

### Description of the changes:
- Add toggle button to switch between the equation and the graph mode
- Only display one mode at a time
- Full support of accessibility
   - high contrast mode
   - Narrator + announcememnt

Also:
- some code cleaning

TODO:
- modify the icon once we receive the final assets.

### How changes were validated:
- manually with light and dark theme + high contrast
- tested with narrator too

### CODE REVIEW

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml [hunk 1]
```diff
@@ -41,22 +211,33 @@
...
         </Grid.ColumnDefinitions>
-
+        <!-- Top panel -->
+        <Grid Grid.ColumnSpan="2">
+            <ToggleButton x:Name="SwitchModeToggleButton" Visibility="Collapsed"
+                          x:Uid="SwitchModeToggleButton"
+                          Checked="SwitchModeToggleButton_Checked"
+                          Unchecked="SwitchModeToggleButton_Checked"
+                          AutomationProperties.AutomationId="SwitchModeToggleButton"
+                          Style="{StaticResource ThemedSwitchModeToggleButtonStyle}"
```
Comment (joseartrivera) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: ThemeResource?

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml [hunk 2]
```diff
@@ -41,22 +211,33 @@
...
+                          ToolTipService.ToolTip="{x:Bind GetInfoForSwitchModeToggleButton(SwitchModeToggleButton.IsChecked.Value), Mode=OneWay}"
+                          AutomationProperties.Name="{x:Bind GetInfoForSwitchModeToggleButton(SwitchModeToggleButton.IsChecked.Value), Mode=OneWay}"
+                          HorizontalAlignment="Right" VerticalAlignment="Stretch" Margin="0,2,4,2">
+                <!-- TODO: update this icon -->
+                <FontIcon Glyph="" FontFamily="Segoe MDL2 Assets" />
```
Comment (joseartrivera) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]:
```suggestion
                <FontIcon Glyph="" FontFamily="{ThemeResource SymbolThemeFontFamily}" />
```

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml [hunk 3]
```diff
@@ -103,25 +282,21 @@
...
-                       RenderTransformOrigin="0,1">
+                <Popup x:Name="TraceValuePopup">
                     <Popup.RenderTransform>
                         <TranslateTransform x:Name="TraceValuePopupTransform" X="50" Y="150"/>
                     </Popup.RenderTransform>
                     <Grid Background="{StaticResource ToolTipBackground}"
                           BorderBrush="{StaticResource ToolTipBorderBrush}"
-                          BorderThickness="3,3,3,3">
+                          BorderThickness="3">
                         <TextBlock x:Name="TraceValue"
```
Comment (joseartrivera) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: This pop up can now go off screen when hovering over the far right side of the graph, we will want to add in some logic to show it to the left in these cases. May want to do this in it's own PR though.

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml [hunk 4]
```diff
@@ -28,11 +30,179 @@
             <Setter Property="BorderThickness" Value="1"/>
             <Setter Property="FontSize" Value="12"/>
         </Style>
-        <converters:BooleanToVisibilityConverter x:Name="BooleanToVisibilityConverter"/>
         <converters:BooleanToVisibilityNegationConverter x:Name="BooleanToVisibilityNegationConverter"/>
+
+        <Style x:Key="SwitchModeToggleButtonStyle" TargetType="ToggleButton">
```
Comment (joseartrivera) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: In high contrast mode the toggled state has a blue background with a blue foreground in High Contrast # 2

Comment (rudyhuyn) [reply src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: This is expected (it's the default style of ToggleButton)

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml (hunk 3 above)
Comment (rudyhuyn) [reply src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: I've replaced the Popup by a lighter control (only a Border instead of a Popup) + move the tooltip to the left or below the pointer when there is not enough space to display it.
